.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
```

Comment:
@MatanAvneri as we have a linter configured, i think it's better to also add eslint job

Reply:
Not exactly how I would like to run it, but we can live with that for now and fix it later